### ISSUE TITLE
Can't read multiple OPC UA nodes in function session.read()

### ISSUE THREAD

**Issue by joselopezbinoovo** [node-opcua/node-opcua#1261]
Hi, I try to read an array of nodeIds with function read(). 

I have two cases. 

In the first case I loop my array of nodesId and pass it to the read function.
The value that it returns is as expected, but in the console I get an alert that I am making many requests, If i increase the array it crash.

```
      const myOpcVariable = await opcUa.findAll()

        const cliet = OPCUAClient.create();
        cliet.on("backoff", (retry, delay) => {
            console.log("Intentado conectar ", endPointOPc, "attempt", retry);
        })

        await client.connect(endPointOPc)
        console.log("Connect to", (endPointOPc));
        const session = await  client.createSession();


        const array = []
        myOpcVariable.forEach(element => {
        array.push(element.dataValues.variableString)
       });
         
/*array[
  'ns=2;s=Simulation_Examples.Functions.Ramp1',
  'ns=2;s=Simulation_Examples.Functions.Ramp2',
  'ns=2;s=Simulation_Examples.Functions.Ramp3',
  'ns=2;s=Simulation_Examples.Functions.Ramp4',
] */
 
        var ArrayplcValues = await Promise.all(array.map( async (nodeId) => {
        let plcValue = await session.read({nodeId:nodeId,AttributeId: AttributeIds.Value,TimestampsToReturn:TimestampsToReturn.Both}); 
        return plcValue.value.value
    })); 

/* console Alert: [NODE-OPCUA-W21] Pending transactions:  ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest ReadRequest
10:10:14.348Z :client_session_impl           :779   [NODE-OPCUA-W22] Warning : your opcua client is sending multiple requests simultaneously to the server ReadRequest    
 ...                                                  please fix your application code */

/* result expected: 
35
150.75
0
-1000
-1000000
0
-1000000000
150.75
*/ 
```

in case 2 I loop my nodeId array and create an array of objects. The problem comes when the result returns on the property AttributeIds: 13 in all the positions of the object.
```
/*array=[  'ns=2;s=Simulation_Examples.Functions.Ramp1',
  'ns=2;s=Simulation_Examples.Functions.Ramp2',
  'ns=2;s=Simulation_Examples.Functions.Ramp3',
  'ns=2;s=Simulation_Examples.Functions.Ramp4', ] */

    let node_to_read=[]
    array.forEach((entry)=> {
     node_to_read.push({nodeId:entry,AttributeId: AttributeIds.Value})
    })
    console.log(node_to_read); 

    let plcValue = await session.read(node_to_read,max_age,function(err,node_to_read,dataValues){
        if (!err) {
            for(var i;i<dataValues.length;i++) {
                   console.log(entry , dataValues[i].toString());
             }
        }
    })
```
In short, I need a solution to be able to read an array of nodeIds in an optimal way.

**Comment by AndreasHeine** [comment]
it looks like you mix async/await syntax with callbacks!?

https://node-opcua.github.io/api_doc/2.32.0/interfaces/node_opcua_client.clientsession.html#read

you can await -> session.read(nodesToRead: [ReadValueIdOptions](https://node-opcua.github.io/api_doc/2.32.0/interfaces/node_opcua.readvalueidoptions.html)[], maxAge?: number): Promise<[DataValue](https://node-opcua.github.io/api_doc/2.32.0/classes/node_opcua.datavalue.html)[]>

returns a Array of DataValue


```ts
const plcValues: DataValue[] = await session.read(node_to_read,max_age)
```
